spec/lib/web_of_science_spec.rb

```diff
@@ -26,4 +26,10 @@
       expect(result).to be_an WebOfScience::Queries
     end
   end
+
+  describe '#logger' do
+    it 'works' do
+      expect(described_class.logger).to be_a Logger
+    end
+  end
```

Comment:
The better spec is to test that it uses the `Settings.WOS.LOG` value, so moving that spec into this spec file from the NotificationManager could resolve the questions on how that changed.